main.c

```diff
@@ -181,9 +184,9 @@ int main(int argc, char *argv[])
 	 * It will have a new stack and an ukarch_ctx
 	 */
 	app_thread = uk_thread_create_container(uk_alloc_get_default(),
```

Comment:
Can we derive all allocator from the scheduler where we attach the thread, to?
`s->a`